app/assets/javascripts/validation.js

```diff
@@ -201,7 +278,7 @@ function validateConfirmPassword(passwordElement, obj) {
     obj.updateUI(
       this,
       false,
-      "Password and Password Confirmation should be the same."
+      "Password and Password Confirmation should be the same"
```

Comment:
Hey! Could you just downcase the words here so it becomes "Password and password confirmation should be the same". It's a minor change but I think it would be useful. Thanks!

Reply:
@SidharthBansal has suggested in title case.

Reply:
Anything else?

Reply:
It is field name, so i said capital letter P